bodhi/server/validators.py

```diff
@@ -1291,6 +1291,18 @@ def validate_from_tag(request: pyramid.request.Request, **kwargs: dict):
     koji_tag = request.validated.get('from_tag')
 
     if koji_tag:
+        # check if any existing updates use this side tag
+        update = request.db.query(Update).filter_by(from_tag=koji_tag).first()
+        if update:
+            if request.validated['edited'] and request.validated['edited'] == update.alias:
```

Comment:
Is ``request.validated`` a dict?
If so, you could simplify this as:
````
if request.validated.get('edited') == update.alias:
````

If there is no `'edited'` in the `request.validated`, it'll be `None` and thus not equal to the alias

Reply:
thanks! commit updated.